docs/troubleshooting.rst

```diff
@@ -10,7 +10,7 @@ Set the logger to log DEBUG logs from django-guid:
 
     LOGGING = {
         'loggers': {
-            'django-guid': {
```

Comment:
The correct logger name is `django_guid`, but we could change it to `django-guid` if you prefer.

Reply:
No, snake_case is correct. :) Its common to name the whole package (on PyPi and git) with dashes, but the code will be with snake_case, true to pep8.